lib/pry-rails/commands/find_route.rb

```diff
@@ -0,0 +1,79 @@
+class PryRails::FindRoute < Pry::ClassCommand
+  match 'find-route'
+  group 'Rails'
+  description 'See which URLs match a given Controller.'
+  command_options argument_required: true
+  banner <<-BANNER
+    Usage: find-route <controller>
```

Comment:
@rf- i tried, it complained about the command not yet being defined (yes i defined the alias outside of the class definition, i have no idea)